pkg/commands/info.go

```diff
@@ -59,9 +59,10 @@ func RunInfo(ctx CommandContext, args InfoArgs) error {
 			if err != nil {
 				return err
 			} else {
-				fmt.Fprintf(ctx.Stdout, "  [%d] %s\n", id, fingerprint)
+				fmt.Fprintf(ctx.Stdout, "  [%d] %s", id, fingerprint)
 			}
 		}
+		fmt.Fprintf(ctx.Stdout, "\n")
```

Comment:
Ok, but why this: https://github.com/scaleway/scaleway-cli/pull/195/files#r40537290